napari_tiff/napari_tiff_colormaps.py

```diff
@@ -0,0 +1,61 @@
+import numpy
+from vispy.color import Colormap
```

Comment:
This import is problematic. But I do not have experience for replacement, I think 
```suggestion
from vispy.color import Colormap
```

Reply:
Problematic how?
Also, your suggested code line currently looks identical to the existing line of code. Did you mean to make a suggestion?

All the colormap stuff is from the original package, so I'm probably going to ask you to make a separate issue if you'd like to see any of that changed. It's out of scope for this PR.